lesson4/homeworklesson4.py

```diff
@@ -0,0 +1,76 @@
+def creating_a_string(first_string:str, second_string:str) -> str:
+    result_string = []
+    second_string = second_string[::-1]
+    if len(first_string) - len(second_string) > 0:
+        n = first_string[-(len(first_string) - len(second_string))::1]
+        for first_string, second_string in zip(first_string, second_string):
+            result_string.append(first_string + second_string)
+        result_string.append(n)
+    elif len(first_string) - len(second_string) < 0:
+        n = second_string[(len(first_string) - len(second_string))::1]
+        for first_string, second_string in zip(first_string, second_string):
+            result_string.append(first_string + second_string)
+        result_string.append(n)
+    else:
+        for first_string, second_string in zip(first_string, second_string):
+            result_string.append(first_string + second_string)
+    result_string = "".join(result_string)
+    return result_string
+
+def sort_list_of_int(test_list:list[int], condition:bool) -> list[int]:
+    for n in range(len(test_list)):
+        for a in range(len(test_list) - n - 1):
+            if test_list[a] > test_list[a + 1]:
+                test_list[a], test_list[a + 1] = test_list[a + 1], test_list[a]
+    if condition == True:
+        return test_list
+    else:
+        return test_list[::-1]
```

Comment:
Но и вообще я бы сделал максимально красиво тогда уже, что-то вроде такого написал бы
```suggestion
    return test_list if not desc else test_list[::-1]
```